### PR TITLE
Feat/integrate iam

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: projects/arlas-toolkit/src/lib/services/startup/startup.service.ts [hunk 1]
```diff
@@ -414,55 +441,104 @@ export class ArlasStartupService {
...
+                headers: {
+                  Authorization: 'bearer ' + authService.accessToken
+                }
+              });
+              // ARLAS-server
+              this.fetchOptions.headers = {
+                Authorization: 'bearer ' + authService.accessToken
+              };
+              // ARLAS-Permission
+              this.permissionService.setOptions({
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/src/lib/services/startup/startup.service.ts:496]: There is a duplicate of the setOptions for the permissionService

**Review by QuCMGisaia — APPROVED** [review]

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 2]
```diff
@@ -97,10 +122,14 @@
 	"Time zone": "Fuseau horaire :",
 	"Remove time filter": "Supprimer le fitre temporel",
 	"User informations": "Informations utilisateur",
-	"Email": "Email",
 	"Organisation": "Organisation",
 	"Roles": "Rôles",
 	"Groups": "Groupes",
+	"Password and Confirm Password must be match": "Les champs Mot de passe et Confirmation doit être identique",
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/fr.json:128]: fixed

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 3]
```diff
@@ -56,18 +56,43 @@
...
 	"Unauthorized action": "Action non autorisée",
 	"You are not authorized to access this resource.": "Vous n'êtes pas autorisé à accéder à cette ressource.",
+	"It’s quick and easy !": "C'est rapide et facile !",
+	"Just enter your email": "Saisissez juste vous email",
+	"By creating an account, you agree to the": "En créant votre compte vous acceptez les",
+	"Terms of Service": "conditions d'utilisation",
+	"User already exists": "L'utilisateur existe déjà",
+	"Registration validated, you will receive an email to validate your account": "Inscription enregistrée, vous allez recevoir un email pour valider votre compte",
+	"Confirm password": "Confirmation",
+	"Your password has been changed.": "Votre mot de passe a été modifié",
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/fr.json:94]: fixed

File: projects/arlas-toolkit/assets/i18n/template.json [hunk 4]
```diff
@@ -56,18 +56,46 @@
...
+	"Email": "Email",
+	"Email is required": "Email is required",
+	"Email must be a valid email address": "Email must be a valid email address",
+	"If you have an account, you will receive an email shortly": "If you have an account, you will receive an email shortly",
+	"Send password reset email": "Send password reset email",
+	"Already have an account?": "Already have an account?",
+	"Log In": "Log In",
 	"interval:": "interval:",
 	"x-axis:": "x-axis:",
+<<<<<<< 3384122b4aa0ea661b88a08af8be9ed9f4f238ea
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/template.json:69]: fixed

File: projects/arlas-toolkit/src/lib/components/forgot/forgot.component.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
+.container {
+  font-family: "Segoe UI", Roboto, "Helvetica Neue", Arial;
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/src/lib/components/forgot/forgot.component.css:2]: fixed

File: projects/arlas-toolkit/src/lib/services/startup/startup.service.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/src/lib/services/startup/startup.service.ts:496]: fixed

**Pull request merged**